app/views/stages/index.html.erb

```diff
@@ -1,13 +1,13 @@
 <%= render 'projects/header', project: @project, tab: "stages" %>
 
 <section class="tabs">
-  <ul id="stages" data-url="<%= reorder_project_stages_path(@project) %>" data-sortable="<%= current_user.is_admin? %>">
+  <ul id="stages" data-url="<%= reorder_project_stages_path(@project) %>" data-sortable="<%= is_user_admin_for_project? %>">
     <% @stages.each.with_index do |stage, index| %>
-      <li class="stage-bar bar list-unstyled clearfix<%= ' sortable' if current_user.is_admin? %>" data-id="stage_id_<%= stage.id %>">
+      <li class="stage-bar bar list-unstyled clearfix<%= ' sortable' if is_user_admin_for_project? %>" data-id="stage_id_<%= stage.id %>">
         <%= link_to stage.name, [@project, stage] %>
         <%= stage_lock_icon stage %>
         <div class="btn-group pull-right">
-          <% if current_user.is_admin? %>
+          <% if current_user.is_admin_for?(@project) %>
```

Comment:
On lines 4 and 6 of this file, you're using the helper method `is_user_admin_for_project?`, but on this line you're saying `current_user.is_admin_for?(@project)`.  Is there a reason you don't want to use the helper method?